changelog/unreleased/flashy-messages-when-loading-folders

```diff
@@ -0,0 +1,6 @@
+Bugfix: Do not display "empty folder" message when there is any content
+
+We've fixed that when some of the list were being loaded, sometimes briefly appeared the "empty folder" message even though the list wasn't empty.
```

Comment:
Proposing: `We've fixed that when some of the file/share lists were being loaded, the "empty folder" message sometimes briefly appeared even though the list wasn't empty.`

Reply:
Done, thanks! :wink: